src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -63,28 +63,28 @@ ov::runtime::Tensor generate(const std::shared_ptr<ngraph::op::v0::Acos>& node,
                              size_t port,
                              const ov::element::Type& elemType,
                              const ov::Shape& targetShape) {
-    return Activation::generate(elemType, targetShape, InputGenerateData(-1, 2, 32768, 1));
+    return Activation::generate(elemType, targetShape, InputGenerateData(2, -1, 32768, 1));
```

Comment:
I suggest to change definition/declaration of InputGenerateData and change order of range/startfrom to startfrom/range.

Looks like it shouldn't cause any issues: grep shows it uses only in generate_inputs.cpp.